pkg/app/instances/gce.go

```diff
@@ -347,3 +363,9 @@ sudo -u vsoc-01 touch /home/vsoc-01/bin/restart_cvd
 printf '#!/bin/bash\nHOME=/var/lib/cuttlefish-common/runtimes /var/lib/cuttlefish-common/artifacts/acloud_link/bin/restart_cvd\n' | sudo -u vsoc-01 tee /home/vsoc-01/bin/restart_cvd >/dev/null
 sudo -u vsoc-01 chmod u+x /home/vsoc-01/bin/restart_cvd
 `
+
+func (m *GCEInstanceManager) GetInfo() (*apiv1.Info, error) {
+	return &apiv1.Info{
+		RuntimeType: "cloud",
```

Comment:
"cloud" is a little ambiguous as it could be AWS, GCP, etc. Even within GCP it could be GCE or GKE.
If you wish to differentiate between "cloud" and "local" or "test", then it would be worth it to add a `name` field to the response and name this one "gce". Or simply have its type be "gce".

Reply:
`RuntimeType` was first introduced to just color runtime cards in Page 0 differently based on some type of runtime.

I think that whether the cloud orchestrator runs from GCP/AWS/etc doesn't have to be shown to Page 0 as far.
If such information should be revealed, we can return `InstanceManagerConfig` without defining new type.

Reply:
Updated: Return `InstanceManagerConfig` as is and **decide the runtime type at Page 0 UI side**  based on instance manager type